src/app/app.component.ts

```diff
@@ -1,10 +1,14 @@
-import { Component } from '@angular/core';
+import { Component, ViewChild, ElementRef, AfterViewInit } from '@angular/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss'],
 })
-export class AppComponent {
-  title = 'bookShop';
+export class AppComponent implements AfterViewInit {
+  @ViewChild ('appTitle')
+  title!: ElementRef;
+  ngAfterViewInit()   {
```

Comment:
add empty line before, fix spacing and add return type

Reply:
fixed